### PR TITLE
BL-5582: added viewModel, template changes, unit tests

### PR DESCRIPTION
**Pull request by raffcecco**

### CODE REVIEW

File: webapp/src/test/java/uk/gov/dvsa/motr/web/viewmodel/EmailConfirmationPendingViewModelTest.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package uk.gov.dvsa.motr.web.viewmodel;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+
+public class EmailConfirmationPendingViewModelTest {
```
Comment (wkainos) [inline webapp/src/test/java/uk/gov/dvsa/motr/web/viewmodel/EmailConfirmationPendingViewModelTest.java]: Hey there's a class `EmailConfirmationPendingResourceTest` you could update `termsTemplateIsRenderedWhenEmailConfirmPendingPathAccessed` test there. 
You can use a code like this to get the viewModel back: `engine.getContext(Map.class).get("viewModel")`.

Comment (raffcecco) [inline webapp/src/test/java/uk/gov/dvsa/motr/web/viewmodel/EmailConfirmationPendingViewModelTest.java]: There were already two viewmodel test classes in /src/test/java/uk/gov/dvsa/motr/web/viewmodel, so I followed that pattern and put my viewmodel test class in there.

Comment (raffcecco) [inline webapp/src/test/java/uk/gov/dvsa/motr/web/viewmodel/EmailConfirmationPendingViewModelTest.java]: Also added a viewModel of the correct class present test in `termsTemplateIsRenderedWhenEmailConfirmPendingPathAccessed`

File: webapp/src/main/resources/template/email-confirmation-pending.hbs [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
-        <p>Click the link in the email to finish signing up for MOT reminders.</p>
+<div class="grid-row">
+    <div class="column-two-thirds">
+        <header class="content-header">
+            <h1 class="content-header__title">
+                One more step
+            </h1>
+        </header>
+        <p class="lede">Check your inbox <strong>{{viewModel.emailDisplayString}}</strong>and click the link in the email from GOV.UK to activate your MOT reminder.</p>
     </div>
```
Comment (Thomas-Franklin) [inline webapp/src/main/resources/template/email-confirmation-pending.hbs]: To me the spacing should be ```inbox <strong>{{viewModel.emailDisplayString}}</strong> and``` as the "and" part will always be required with a space.

**Comment by Thomas-Franklin** [comment]
Please squash the commits as additional tests were added by me and @Jerotire

File: webapp/src/test/java/uk/gov/dvsa/motr/web/resource/EmailConfirmationPendingResourceTest.java [hunk 3]
```diff
@@ -1,28 +1,70 @@
...
+    @Before
+    public void setup() {
+
+        motrSession = mock(MotrSession.class);
+        engine = new TemplateEngineStub();
+        resource = new EmailConfirmationPendingResource(engine, motrSession);
+    }
+
+    @After
+    public void clearDown() {
```
Comment (aaronlong) [inline webapp/src/test/java/uk/gov/dvsa/motr/web/resource/EmailConfirmationPendingResourceTest.java]: I don't think this clearDown method is required.  As in the setup method a new mock is being created for the MotrSession.

Comment (Thomas-Franklin) [inline webapp/src/test/java/uk/gov/dvsa/motr/web/resource/EmailConfirmationPendingResourceTest.java]: yea true, it wouldn't be :D my bad - was doing it as I was leaving

**Pull request merged**